.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: scala
 sudo: false
 scala:
-  - 2.11.12
+  - 2.12.7
```

Comment:
do we have to the same version as of spark release or can we use 2.12.8 ?

Reply:
In theory at least, we should be able to use 2.12.8. I tried it out locally and it passed the tests. I'll push the change.